testdata/p4_16_dpdk_errors_outputs/pna-dpdk-direct-meter-err-1.p4-error

```diff
@@ -1,6 +1,6 @@
-[--Werror=unexpected] error: execute method for different DirectMeter instances (meter0_0 and meter1_0) called within same action
-[--Werror=unexpected] error: execute method for different DirectMeter instances (meter0_0 and meter1_0) called within same action
```

Comment:
Is it correct that these error messages are no longer present?  Were they only present before because of some bug that you fixed with this PR?

Reply:
Yes. The PR does not seem to change anything that would trigger this error output change.